source/extensions/filters/network/mongo_proxy/proxy.cc

```diff
@@ -366,11 +366,11 @@ void ProxyFilter::onEvent(Network::ConnectionEvent event) {
   }
 
   if (event == Network::ConnectionEvent::RemoteClose && !active_query_list_.empty()) {
-    stats_.cx_destroy_local_with_active_rq_.inc();
+    stats_.cx_destroy_remote_with_active_rq_.inc();
```

Comment:
It would be good to add a release note to the "Minor Behavior Changes" section since this change affects monitoring behavior.

Release notes can be found at: docs/root/version_history/current.rst